.travis.yml

```diff
@@ -0,0 +1,83 @@
+language: cpp
+
+os: linux
+
+compiler: gcc
+
+sudo: true
+
+env:
+  GNAT: "${HOME}/opt/gnat"
+  CBMC: "${HOME}/cbmc"
+
+cache: ccache
```

Comment:
If you look at how we handle ccache on, say, the CBMC repo you'll see theres a couple of additional things we'll want to do:
1) Specify a limit on the size of the ccache - I think for CBMC we set it to 1GB (default is something like 5GB otherwise) - this is important because over time the size of the cache will grow, and that then slows down travis downloading/uploading its cache at the beginning/end of the job
2) Its useful to zero ccaches' stats (not contents, just the counters) at the beginning of the job (`ccache -z`), and then print the ccache counters at the end of the job (`ccache -s`) - this enables us to see how effectively the ccache is working.

Reply:
Oh, and 3) - make sure that the cbmc build is actually calling ccache, and not just calling g++... Again, check the CBMC travis job to see how that does this (IIRC it does this explicitly)